front/src/assets/css/tokens.css

```
@@ -2,25 +2,25 @@
 
 :root {
   /* Main colors (HSL) */
-  --color-primary: 274, 36%, 43%; /* Royal Twilight */
-  --color-primary-light: 274, 36%, 70%; /* Pale Lightened Violet */
-  --color-primary-dark: 274, 42%, 23%; /* Dark Violet */
-  --color-primary-soft: 274, 36%, 95%; /* Soft Violet Background */
-
-  --color-accent: 330, 80%, 52%; /* Electric Raspberry */
-  --color-accent-light: 330, 80%, 75%; /* Light Pink */
-  --color-accent-dark: 330, 80%, 30%; /* Dark Raspberry */
-  --color-accent-soft: 330, 80%, 96%; /* Soft Pink Background */
-
-  --color-grey-100: 280, 25%, 98%; /* Very Light Grey */
-  --color-grey-200: 270, 20%, 92%; /* Light Grey */
-  --color-grey-300: 270, 15%, 85%; /* Medium Grey */
-  --color-grey-400: 270, 10%, 70%; /* Strong Grey */
-  --color-grey-900: 270, 10%, 15%; /* Almost Black Grey */
+  --color-primary: hsl(274deg 36% 43%); /* Royal Twilight */
+  --color-primary-50: hsl(from var(--color-primary) h s l / 50%); /* Ultra Light Violet */
```

Pour gérer l'opacité je te conseille d'utiliser hsla, ce sera plus simple. (c'est comme rgba pour rgb, ça rajoute un paramètre d'opacité.

```suggestion
  --color-primary-50: hsla(var(--color-primary-light), 0.5); /* Ultra Light Violet */
```

https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Values/color_value/hsl